autopush/db.py

```diff
@@ -1,6 +1,7 @@
 """Database Interaction"""
 import uuid
 from functools import wraps
+from base64 import urlsafe_b64decode, urlsafe_b64encode
```

Comment:
Because that's how the headers are encoded, and I wanted the client to use the same decoder for both. :wink: (It's easy to support URL-safe and regular; just thought I'd keep it consistent).